Remove incorrect ASSERT in RemoteLayerTreeLayers

grorg opened the pull request:
#### 79921d502bc7608608baa916f670bc84419b590f
<pre>
Remove incorrect ASSERT in RemoteLayerTreeLayers
<a href="https://bugs.webkit.org/show_bug.cgi?id=243440">https://bugs.webkit.org/show_bug.cgi?id=243440</a>
&lt;rdar://problem/97953495&gt;

Reviewed by Tim Horton.

RemoteLayerTrees was calling an unknown method in an ASSERT within
CG_DISPLAY_LIST_BACKED_IMAGE_BUFFER.

* Source/WebKit/UIProcess/RemoteLayerTree/cocoa/RemoteLayerTreeLayers.mm:
(-[WKCompositingLayer _setWKContents:withDisplayList:replayForTesting:]):

Canonical link: <a href="https://commits.webkit.org/253049@main">https://commits.webkit.org/253049@main</a>
</pre>

hortont424 commented on Source/WebKit/UIProcess/RemoteLayerTree/cocoa/RemoteLayerTreeLayers.mm:
```diff
@@ -57,7 +57,6 @@ - (void)_setWKContents:(id)contents withDisplayList:(WebKit::CGDisplayList&&)dis
 
     if (replay) {
         _displayListDataForTesting = data;
-        ASSERT(displayList.ports.isEmpty());
```
This is fine, since now we only have takeSurfaces(), no way to peek at them. Not an important assertion, anyway.

grorg committed:
Remove incorrect ASSERT in RemoteLayerTreeLayers
https://bugs.webkit.org/show_bug.cgi?id=243440
<rdar://problem/97953495>

Reviewed by Tim Horton.

RemoteLayerTrees was calling an unknown method in an ASSERT within
CG_DISPLAY_LIST_BACKED_IMAGE_BUFFER.

* Source/WebKit/UIProcess/RemoteLayerTree/cocoa/RemoteLayerTreeLayers.mm:
(-[WKCompositingLayer _setWKContents:withDisplayList:replayForTesting:]):

Canonical link: https://commits.webkit.org/253049@main
